alert/alerts.py

```diff
@@ -47,11 +47,15 @@ def __init__(self, reg):
         super().__init__('alert/email_alert.html', reg)
 
     def send_alert(self):
-        if self.registration.email is None:
+        if self.registration.user.profile.email is not None:
```

Comment:
This might cause an error if `self.registration.user` is None -- would add change the line to `if self.registration.user is not None and self.registration.user.profile.email is not None` -- there may be a more pythonic way to do that, but it's tough since user is nullable, profile is nullable, and email is nullable.